config/envs/default.py

```diff
@@ -45,3 +45,94 @@ class DefaultConfig:
     FUND_ENDPOINT = "/funds/{fund_id}"
     FUND_ROUNDS_ENDPOINT = "/funds/{fund_id}/rounds"
     FUND_ROUND_ENDPOINT = "/funds/{fund_id}/rounds/{round_id}"
+
+    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    # COF FORMS BASE CONFIG
+    COF_R2_FORMS = [
+        {
+            "form_minting_name": "applicant-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "asset-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-benefits",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-engagement",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-representation",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "community-use",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "declarations",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "environmental-sustainability",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "feasibility",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "funding-required",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "inclusiveness-and-intergration",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "local-support",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "organisation-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-costs",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-information",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "project-qualification",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "risk",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "skills-and-resources",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "value-to-the-community",
+            "questions": [],
+        },
+        {
+            "form_minting_name": "upload-business-plan",
+            "questions": [],
+        },
+    ]
+
+    FUND_ROUND_FORMS = {
+        "community-ownership-fund:round-2": COF_R2_FORMS.copy(),
```

Comment:
Rather than doing a colon separated string, can we use either a dictionary or separate values for the round and fund? This would make it easier to use.

Reply:
I will implement this in fs-1340 if that's ok?

Reply:
I pushed a little change so that this works with the docker runner properly: https://github.com/communitiesuk/funding-service-design-application-store/commit/94b678fbbcc7eb2a411510c743acb9c373c6b4ff